### ISSUE TITLE
Add a TodayLine config that would display a vertical line on an x-axis filled with javascript dates.

### ISSUE THREAD

**Issue by gotwilk890** [swimlane/ngx-charts#1407]
**Is your feature request related to a problem? Please describe.**
Would love it if we could somehow display a today line on these graphs. Specifically a vertical one for the line graph. My x-axis are javascript dates.

**Describe the solution you'd like**
Have a config for todayLine: true that automatically places a today line if it fits within the date range.

**Describe alternatives you've considered**
I could potentially create a custom chart to handle this. But it feels like overkill.

**Additional context**

**Comment by Moulde** [comment]
Yes, I would love this as well - In my case it is to seperate days from each other.
